contribs/gmf/src/directives/print.js

```diff
@@ -661,32 +675,22 @@ gmf.PrintController.prototype.isAttributeInCurrentLayout_ = function(name) {
 
 
 /**
- * Getter setter to update or get the current rotation value. Param and result
- *     are in degree. Updating the rotation will redraw the mask or rorate the
- *     map (depending on the configuration);
- * @param {number=} opt_rotation The optional new rotation value.
- * @return {number} The new value of rotation;
- * @export
+ * Set the current rotation value.
+ * Updating the rotation will redraw the mask or rotate the map (depending on the configuration).
+ * @param {number} rotation The optional new rotation value in degrees.
  */
-gmf.PrintController.prototype.getSetRotation = function(opt_rotation) {
-  if (opt_rotation !== undefined) {
-    let rotation = parseInt(opt_rotation, 10);
-    if (rotation > 180) {
-      rotation = -180;
-    } else if (rotation < -180) {
-      rotation = 180;
-    }
-    this.rotation = rotation;
+gmf.PrintController.prototype.setRotation = function(rotation) {
+  this.rotation = ol.math.clamp(rotation, -180, 180);
```

Comment:
I don't know this function, are you sure it is supported by our browser we support ?

Reply:
This function is from OpenLayers, see:
https://github.com/openlayers/openlayers/blob/v4.6.5/src/ol/math.js#L14

Reply:
Agree, thanks